package.json

```diff
@@ -45,7 +47,10 @@
   },
   "main": "lib/protractor.js",
   "scripts": {
-    "pretest": "./scripts/pretest.sh",
+    "prepublish": "./node_modules/.bin/typings install && ./node_modules/typescript/bin/tsc && cp built/*.js lib/",
```

Comment:
Would this override what's currently in lib? I think it would be cleaner to just move everything we need into `built` and publish from there.